modules/shortcodes/slideshow.php

```diff
@@ -182,6 +183,17 @@ public function shortcode_callback( $attr ) {
 			);
 		}
 
+		if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+			// Load the styles and use the rendering method from the Slideshow block.
+			Jetpack_Gutenberg::load_styles_as_required( 'slideshow' );
```

Comment:
Will the `Jetpack_Gutenberg` class always be available?

Reply:
`Jetpack_Gutenberg` will be available, yes. 👍

Reply:
OK, nice!